src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -108,14 +114,18 @@ public class LogManagerService extends PluginService {
     public static final String UPLOAD_TO_CW_CONFIG_TOPIC_NAME = "uploadToCloudWatch";
     public static final String MULTILINE_PATTERN_CONFIG_TOPIC_NAME = "multiLineStartPattern";
     public static final int DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC = 300;
+    public static final int DEFAULT_MAX_FILES_TO_TRACK_PER_COMPONENT = 5;
     private final Object spaceManagementLock = new Object();
     private final List<Consumer<EventType>> serviceStatusListeners = new ArrayList<>();
 
     // public only for integ tests
     public final Map<String, Instant> lastComponentUploadedLogFileInstantMap =
             Collections.synchronizedMap(new LinkedHashMap<>());
-    final Map<String, CurrentProcessingFileInformation> componentCurrentProcessingLogFile =
+
+    public final Map<String, ProcessingFileLRU> processingFilesInformation =
```

Comment:
nit: processingFilesInformation -> processingFileLRUs. To do that just for differing from the following name [here](https://github.com/aws-greengrass/aws-greengrass-log-manager/blob/cea1473c4dcc232447f1665f358dc4fd6f289de0/src/main/java/com/aws/greengrass/logmanager/LogManagerService.java#L431)

Reply:
This is called processingFilesInformation (plural)